src/components/AuthorsCards/styled.ts

```diff
@@ -0,0 +1,166 @@
+import styled from 'styled-components'
+
+// AuthorsCards images
+import jindrichmaca from '../../assets/AuthorsCards/jindrichmaca.jpg'
+import janachaloupkova from '../../assets/AuthorsCards/janachaloupkova.jpg'
+import martinavilimova from '../../assets/AuthorsCards/martinavilimova.jpeg'
+import tomashampl from '../../assets/AuthorsCards/tomashampl.jpg'
+
+export const Wrapper = styled.section`
+	display: grid;
+	grid-template-columns: 2fr 3fr 3fr 2fr;
+	grid-gap: 1.5em;
+	grid-template-areas:
+		'. title title .'
+		'. card1 card2 .'
+		'. card3 card4 .';
+	justify-content: stretch;
```

Comment:
Grid-area I think it has a good application in more complex web layouts. Here it would be enough to give a flex that takes up space first 50% and then 100%. There would be less code and it would be easier to edit.

Reply:
Fixed! Flex wasn't possible with this page layout, I would have to add one more div to the mix. Instead, I deleted the grid area (I only did it to try it out lol) and made every :nth() child of the Card element either first or second column. It seemed clearer since it didn't needed the extra div.
![grid2](https://user-images.githubusercontent.com/76131194/108770655-de2e1300-755a-11eb-982c-75fa5744db1d.png)
![grid1](https://user-images.githubusercontent.com/76131194/108770658-dec6a980-755a-11eb-95f0-312722f159bb.png)